optimizely/config/polling_manager.go

```diff
@@ -114,10 +112,10 @@ func NewPollingProjectConfigManagerWithOptions(exeCtx utils.ExecutionCtx, sdkKey
 
 	pollingProjectConfigManager := PollingProjectConfigManager{requester: requester, pollingInterval: pollingInterval, exeCtx: exeCtx}
 
-	pollingProjectConfigManager.activate(options.Datafile, true) // initial poll
+	pollingProjectConfigManager.start(options.Datafile, true) // initial poll
 
 	cmLogger.Debug("Polling Config Manager Initiated")
-	go pollingProjectConfigManager.activate([]byte{}, false)
+	go pollingProjectConfigManager.start([]byte{}, false)
```

Comment:
This looks awkward that we call start twice. I did not change this right away, but we should change this behavior.